assets/espresso/bands.in

```diff
@@ -0,0 +1,5 @@
+&BANDS
+{%- for key, value in &BANDS %}
```

Comment:
Does jinja/nunjucks allow variable names starting with `&` ?

Reply:
Yeah, we should not have variables starting with `&`